### PR TITLE
[4.0] Extensions Manage

### PR DESCRIPTION
**Pull request by brianteeman**
Moved the tooltip with the description of the extension to an accessible div and added a tabindex so that this can be accessed by keyboard users.

Removed the author tooltip as I don't see the need to display that information (can always be put back)

### After
image

### CODE REVIEW

File: administrator/components/com_installer/tmpl/manage/default.php [hunk 1]
```diff
@@ -132,9 +133,7 @@
 									<?php echo isset($item->creationDate) && $item->creationDate !== '' ? $item->creationDate : '&#160;'; ?>
 								</td>
 								<td class="d-none d-md-table-cell">
-									<span class="editlinktip hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', Text::_('COM_INSTALLER_AUTHOR_INFORMATION'), $item->author_info, 0); ?>">
-										<?php echo isset($item->author) && $item->author !== '' ? $item->author : '&#160;'; ?>
-									</span>
+									<?php echo isset($item->author) && $item->author !== '' ? $item->author : '&#160;'; ?>
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/manage/default.php]: Shorten to `!empty` instead? If yes, do the same for line 133.

Comment (brianteeman) [reply administrator/components/com_installer/tmpl/manage/default.php]: like this?

`<?php echo !empty($item->author) ? $item->author : '&#160;'; ?>`

Comment (Quy) [reply administrator/components/com_installer/tmpl/manage/default.php]: Yes please.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 609ce9e4d5d7f7b377b80de3abef213e1b69102d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25226">issues.joomla.org/tracker/joomla-cms/25226</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 609ce9e4d5d7f7b377b80de3abef213e1b69102d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25226">issues.joomla.org/tracker/joomla-cms/25226</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by roland-d** [comment]
Thank you.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**